src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -1071,6 +1072,7 @@ environments:
         * storage_min_free_bytes: 10485760 (10MiB)
         * topic_partitions_per_shard: 1000
         * fetch_reads_debounce_timeout: 10
+		* group_initial_rebalance_delay: 0
```

Comment:
I think the lines above use spaces, whereas this uses two tabs

Reply:
yikes, didn't look this way in the IDE. fixed.